lib/Client.js

```diff
@@ -1504,7 +1512,7 @@ var Client = (function () {
 							for (var _iterator14 = self.pmChannelCache[Symbol.iterator](), _step14; !(_iteratorNormalCompletion14 = (_step14 = _iterator14.next()).done); _iteratorNormalCompletion14 = true) {
 								var pmc = _step14.value;
 
-								if (pmc.user.equals(destination)) {
+								if (pmc.user && pmc.user.equals(destination)) {
```

Comment:
When the client lost contact with another user (leave server) the channel is keep alive and the PMChannel.user is set to null.